docs/docs/plugins/proxy-floating-layout.md

```diff
@@ -1,10 +1,10 @@
-# Floating Layout Plugin
+# Proxy Floating Layout Plugin
```

Comment:
Agree.
I have a question related to naming.
Should we keep `ProxyFloatingLayoutCommands` or should we rename it `FloatingLayoutPluginCommands` ?
And keep the proxy only for the internal engine.

Reply:
I think the naming should be consistent with the plugin name.

However, this made me think that there's a risk of confusion between the two types of engines. Maybe this would be alleviated by making the plugin something akin to `GlobalFloatingPlugin` and adding the `global` word to the commands.

Reply:
Okey, I tried to rename things.

The `ProxyFloatingLayoutPlugin` was renamed `FloatingWindowPlugin`.
Should I add an `s`?  `FloatingWindowsPlugin`?

I adapted the identifier of the `FloatingWindowCommands` by changing the identifier from `whim.floating_layout` to `whim.floating_window`. I don't know if the identifier prefix must be aligned with the C# package.
Doing this way, it leaves open the `whim.floating_layout` if we want to add custom command for that specific layout.